Resources/Prototypes/Nyanotrasen/Entities/Structures/Furniture/Barrels/barrels.yml

```diff
@@ -0,0 +1,62 @@
+- type: entity
+  id: BarrelBase
+  parent: BaseStructure
+  name: barrel
+  description: You feel a great urge to roll it down a hill. 
+  abstract: true
+  components:
+  - type: Anchorable
+  - type: Transform
+    noRot: true
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: PlaceableSurface
+
+- type: entity
+  id: BarrelSteel
+  parent: BarrelBase
+  name: steel barrel
+  components:
+  - type: Sprite
+    sprite: Nyanotrasen/Structures/Furniture/barrels.rsi
+    state: steel
+  - type: Icon
+    sprite: Nyanotrasen/Structures/Furniture/barrels.rsi
+    state: steel
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 125
+      behaviors:
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 25
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          path: /Audio/Effects/metalbreak.ogg
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          SheetPlasteel1:
+            min: 5
+            max: 5
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+        
+- type: entity
+  id: BarrelExplosive
+  parent: BarrelSteel
+  name: explosive barrel
+  description: On further inspection, it's not explosive. 
```

Comment:
We can probably repurpose this into an oil drum at some point, if these barrels prove useful.